webpack/containers/Header.js

```diff
@@ -9,9 +9,18 @@ import NotificationDropdown from '../components/NotificationDropdown';
 import NotificationMenu from '../components/NotificationMenu';
 import { fetchNotifications } from '../actions/notification_actions';
 
-let LoginMenu = ({logInOpener, signUpOpener, currentUser}) => {
+let LoginMenu = ({openIdServer, logInOpener, signUpOpener, currentUser}) => {
   let loggedIn = ! _.isEmpty(currentUser);
   if(!loggedIn) {
+    if(openIdServer != '') {
+      return (
+        <ul className="nav navbar-nav">
+          <li className="log-in-link">
+            <a tabIndex="2" href={openIdServer}>Login</a>
```

Comment:
It appears that we are going to link directly to whatever gets passed in. It is originally a global variable called OPIDC_HOST, which would imply it is a hostname. It is then passed into openIdServer, which is a different name. I would recommend changing the global to OIDC_LOGIN_URL and then the component prop could be oidcLoginURL.